dev-python/pefile/pefile-2022.5.30-r1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{8..10} )
+inherit distutils-r1
+
+DESCRIPTION="Module to read and work with Portable Executable (PE) files"
+HOMEPAGE="https://github.com/erocarrera/pefile"
+
+if [[ ${PV} == 9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/erocarrera/pefile.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/erocarrera/pefile/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+
+PATCHES=(
+	"${FILESDIR}/${P}-remove-future.patch"
```

Comment:
Can you fetch it from GitHub (append `.patch` to the commit URL) or is that not clean to apply? The patch is a big large-ish.

Reply:
It's not clean to apply. I modified the patch. There is a change in the `tests`-directory which is not shipped in the release tarball.